compositor_common/src/scene/builtin_transformations.rs

```diff
@@ -1,11 +1,16 @@
 use serde::{Deserialize, Serialize};
 
-use crate::util::RGBAColor;
+use crate::util::{Coord, RGBAColor};
 
 #[derive(Serialize, Deserialize, Clone)]
 #[serde(tag = "transformation", rename_all = "snake_case")]
-pub enum BuiltinTransformation {
+pub enum BuiltinTransformationSpec {
```

Comment:
I know that in a previous PR, @wkozyra95 suggested removing the `spec` suffix, but while writing this PR I found out, that it's more straightforward to use the `spec` suffix for user-provided / deserialized specifications. 
Without the prefix, it was hard to distinguish between the parameters of transformation (provided in spec) and GPU pipeline (shader transformation).

Reply:
I think my comment was for user facing values, so I think this is fine in type names, but not in the field names. `BuiltinTransformationSpec` is fine, but e.g. `textures_spec` below I would rename.

btw when I'm talking about public api I'm usually talking about json format not the rust types we are exporting